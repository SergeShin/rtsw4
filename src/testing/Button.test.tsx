import * as React from "react";
import { shallow } from "enzyme";
import Button from "./Button";

test("check", () => {
	const element = shallow(<Button>Sample Button</Button>);

	expect(element.text()).toEqual("Sample Button");

	expect(element.find("button").hasClass("red")).toEqual(true);

	element.find("button").simulate("click");

	expect(element.find("button").hasClass("green")).toEqual(true);

	expect(shallow).toMatchSnapshot();
});
