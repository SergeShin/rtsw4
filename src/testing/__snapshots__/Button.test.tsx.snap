// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check 1`] = `[Function]`;
